app/assets/javascripts/components/user_components/user_overview_info.js.jsx

```diff
@@ -0,0 +1,26 @@
+function Info(props) {
+        return (
+            <div className={"row col-sm-10"}>
+                <div className={"card-text col-sm-2"}><h6>Full name:</h6></div>
```

Comment:
try to build classes in order to override w/o `!important`
e.g.:
first bootstrap classes, then yours

```
instead of 
"cart-text col-sm-2"
to
"col-sm-2 cart-text"
```